src/acceptance_tests/acceptance_tests_suite_test.go

```diff
@@ -77,10 +77,8 @@ func setBoshEnvironmentVariables() {
 	director, err := buildDirector(directorIP)
 	Expect(err).NotTo(HaveOccurred())
 
-	info, err := director.Info()
+	_, err = director.Info()
 	Expect(err).NotTo(HaveOccurred())
-	Expect(info.Name).To(Equal("docker"))
```

Comment:
thanks. Was there a special change from bosh colleagues that now these vars are no longer there ?

Reply:
We updated the BOSH CPI that runs this test from the docker one to the warden one. That's why this check doesn't work anymore.